From de3174aa1d1293636079473ff693d9a274a093a9 Mon Sep 17 00:00:00 2001
From: Heiko Becker <heiko.becker@kde.org>
Date: Fri, 16 Dec 2022 00:01:15 +0100
Subject: [PATCH] Add missing include

This fixes the build with boost 1.81.0. <array> was included by one of
the boost headers, however, it's no longer included as of Boost 1.81.0.
---
 src/analyze/gui/histogrammodel.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/analyze/gui/histogrammodel.h b/src/analyze/gui/histogrammodel.h
index df11fa3..49b9a4e 100644
--- a/src/analyze/gui/histogrammodel.h
+++ b/src/analyze/gui/histogrammodel.h
@@ -9,6 +9,8 @@
 
 #include <QAbstractTableModel>
 
+#include <array>
+
 #include "treemodel.h"
 
 struct HistogramColumn
-- 
2.39.1

